src/Entity/User.php

```diff
@@ -134,4 +134,13 @@ public function setPlainPassword(string $plainPassword): User
         $this->plainPassword = $plainPassword;
         return $this;
     }
+
+    /**
+     * toString
+     * @return string
+     */
+    public function __toString()
+    {
+        return $this->getUserName();
```

Comment:
You are in the class, you can access the property directly, there is no point to use the getter here